### PR TITLE
Clinic database tables

### PR DESCRIPTION
**Pull request by rolandfbk**
My partners and I created the Clinic database tables according to the project requirements. Below is the list of the tasks that we implemented:

- Created patients, medical_histories, treatments, invoices, and invoice_items tables
- Set foreign keys on medical_histories, invoices, invoice_items, and invoice_items
- Created a many-to-many table called"bill", to link the invoices and treatments tables
- Created Indexes on patient_id, medical_histories_id, invoice_id, and treatment_id attributes

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ALTER TABLE invoices ADD CONSTRAINT fk_invoices_medical_history FOREIGN KEY (medical_histories_id) REFERENCES medical_histories(id);
+
+ALTER TABLE invoice_items ADD CONSTRAINT fk_invoice_items_invoices FOREIGN KEY (invoice_id) REFERENCES invoices(id);
+
+ALTER TABLE invoice_items ADD CONSTRAINT fk_invoice_items_treatment FOREIGN KEY (treatment_id) REFERENCES treatments(id);
+
+CREATE TABLE bills (
+  invoice_id INT,
+  treatment_id INT
+);
```
Comment (gemmen29) [inline schema_based_on_diagram.sql]: - [ ] Please remove the `bills` table as there is no relation between `invoices` and `treatments`  in the diagram
- [ ] Kindly create a table to represent the relation between `medical_histories` and `treatment` as the relationship is many to many. don't forget to create indexes for the foreign keys in that table.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — APPROVED** [review]
Hi @rolandfbk, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by braincee** [comment]
Thanks very much, @gemmen29 @Lordkaito 

catthankyou-cat

**Pull request merged**